wagtail/images/models.py

```diff
@@ -403,16 +404,23 @@ def run(self, image, output):
                 # Developer specified an output format
                 output_format = env['output-format']
             else:
-                # Default to outputting in original format
-                output_format = original_format
-
-                # Convert BMP files to PNG
-                if original_format == 'bmp':
-                    output_format = 'png'
+                # Convert bmp and webp to png by default
+                default_conversions = {
+                    'bmp': 'png',
+                    'webp': 'png',
+                }
 
                 # Convert unanimated GIFs to PNG as well
-                if original_format == 'gif' and not willow.has_animation():
-                    output_format = 'png'
+                if not willow.has_animation():
+                    default_conversions['gif'] = 'png'
+
+                # Allow the user to override the conversions
+                conversion = getattr(settings, 'WAGTAILIMAGES_FORMATS', {})
```

Comment:
Should probably be `WAGTAILIMAGES_FORMAT_CONVERSION` as images already uses format(s)